src/utils/validators.js

```diff
@@ -116,6 +117,13 @@ export const getWalletNameError = (
   }
 }
 
+export const getUnstoppableDomainAddress = async (domain: string): Promise<Address> => {
+  const resolution = new Resolution();
+  try {
+    return await resolution.addr(domain, 'ADA')
+  } catch(e) { return domain; } // invalid domain
```

Comment:
You may want to handle errors according to error types. Please take a look at the following article (errors handling section): https://docs.unstoppabledomains.com/send-and-receive-crypto-payments/crypto-payments#error-handling

Reply:
Understood, 0xjim brought this up with me and I have uncommitted, almost ready, changes that evaluate each error state.
Currently I am waiting on the Yoroi team's decisions around design but can commit these changes if you wish.

Reply:
Resolved in [297c3b1754a64ce4b81e5739db428adc357a1a18](https://github.com/Emurgo/yoroi-mobile/pull/1465/commits/297c3b1754a64ce4b81e5739db428adc357a1a18)

Reply:
Resolved in [5ed3f543e0c77e57b59ec3fbd3caadf69b2c621b](https://github.com/Emurgo/yoroi-mobile/pull/1465/commits/5ed3f543e0c77e57b59ec3fbd3caadf69b2c621b)